development/webpack/utils/helpers.ts

```diff
@@ -232,3 +232,16 @@ export function logStats(err?: Error | null, stats?: Stats) {
  * @returns a new array with duplicate values removed and sorted
  */
 export const uniqueSort = (array: string[]) => [...new Set(array)].sort();
+
+/**
+ * Generates a runtime URL expression for a given path.
+ *
+ * This function constructs a URL string using the `runtime.getURL` method
+ * from either the `globalThis.browser` or `chrome` object, depending on
+ * which one is available in the global scope.
+ *
+ * @param path - The path of the runtime URL.
+ * @returns The constructed runtime URL string.
+ */
+export const getRuntimeURLExpression = (path: string) =>
```

Comment:
Can we put this function in the `SelfInjectPlugin` directory? I think putting it in the `index.ts` file is fine, since there aren't any helpers there. The reason is that I'd like to try to keep the plugin's logic isolated.